src/ImageRunModal.jsx

```diff
@@ -423,7 +423,7 @@ export class ImageRunModal extends React.Component {
                             } else {
                                 dialogError = _("Failed to search for new images");
                                 // TODO: add registry context, podman does not include it in the reply.
-                                dialogErrorDetail = cockpit.format(_("Failed to search for images: $0"), result.reason);
+                                dialogErrorDetail = result.reason ? cockpit.format(_("Failed to search for images: $0"), result.reason.message) : _("Failed to search for images.");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1172-20230106-100359-1eb6bb49-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)